src/css/_booking-form.scss

```diff
@@ -6,13 +22,42 @@
     padding: 10px;
     border-radius: 10px;
   }
-
-  // button {
-  //   @import buttonStyling(); 
-  //   background-color: $bold-color4;
-  //   color: $muted-color1;
-  // }
 }
 
+.booking {
+  background-color: $bold-color3;
+  display: flex;
+  flex-direction: column;
+  position: fixed;
+  // bottom: 0;
+  // right: 0;
+  justify-content: center;
+  height: auto;
+  width: 20vh;
+  height: 80vh;
+  padding: 2em;
+
+  select {
+    border-radius: 10px;
+    margin: 10px;
```

Comment:
If you're trying to make your borders more responsive, consider changing your margins and paddings to percentages, rems, ems, or vw/vh.